README.md

```diff
@@ -1,13 +1,21 @@
 # MySQL Driver
 
-### See [issue #1](https://github.com/mattes/migrate/issues/1#issuecomment-58728186) before using this driver!
-
 * Runs migrations in transactions.
   That means that if a migration fails, it will be safely rolled back.
 * Tries to return helpful error messages.
-* Stores migration version details in table ``schema_migrations``.
+* Stores migration version details in table `schema_migrations`.
   This table will be auto-generated.
+* Safe to run concurrently (`schema_migrations` table is locked during migrations)
+
+## Migrations SQL formatting
 
+Each SQL statement MUST end with semicolon (;) FOLLOWED BY NEWLINE !
+Whole migration will be executed inside transaction by default.
+Place SQL between "-- TXBEGIN" and "-- TXEND" comments for custom transaction:
+  - you CAN have multiple separate transactions in single migration
+  - any SQL not wrapped into TXBEGIN - TXEND will be executed without transaction.
+Add "-- NOTX" comment above all SQL to disable default transaction. NOTE:
```

Comment:
We should probably unify this with the existing `-- disable_ddl_transaction` in pg driver

Reply:
I think it's better to keep this "directive" comments short to avoid potential misspelling, or we should add some prefix (like `++` in goose) to apply validations and throw error if user misspelled something